src/bh1750/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ upm_mixed_module_init (NAME bh1750
     CPP_SRC bh1750.cxx
     FTI_SRC bh1750_fti.c
     CPP_WRAPS_C
-    REQUIRES upmc-utilities mraa)
+    REQUIRES utilities-c mraa)
```

Comment:
As mentioned above, specifying utilities-c is no longer needed here any more right?  I won't comment on any further instances regarding this issue.  And of course, mraa is, if mraa is actually used by the driver?